### DISCUSSION TITLE
Disable nvim-cmp Up and Down selection

### DISCUSSION THREAD

**Discussion by ghost** [LunarVim/LunarVim#4367] [Q&A]
I would like to disable the completion selection with Up and Down arrows and leave only Tab/Shift-Tab selection, is it possible to achieve that through `lvim.builtin.cmp`?

I tried `lvim.builtin.cmp.mapping['Up'] = nil` but it didn't work; after this, I tried commenting out these two keys:
![grafik](https://github.com/LunarVim/LunarVim/assets/142456149/a8a4bf3e-5d2e-4b69-b370-c9c7cb419229)

to no avail.

**Comment by juicy-g** [comment]
This is what I came up with:
```lua
local cmp_mapping = require 'cmp.config.mapping'
local cmp = require('lvim.utils.modules').require_on_index 'cmp'

lvim.builtin.cmp.mapping['<Up>'] = cmp_mapping({
  i = function(fallback)
    if cmp.visible() then
      cmp.close()
    end
    fallback()
  end
})

lvim.builtin.cmp.mapping['<Down>'] = cmp_mapping({
  i = function(fallback)
    if cmp.visible() then
      cmp.close()
    end
    fallback()
  end
})
```
This snippet will close the CMP window and allow the cursor to move up and down as per normal in insert mode.
